fabric-data-generation-api-v1/src/main/resources/fabric.mod.json

```diff
@@ -28,6 +28,11 @@
   ],
   "accessWidener" : "fabric-data-generation-api-v1.accesswidener",
   "custom": {
-    "fabric-api:module-lifecycle": "stable"
+    "fabric-api:module-lifecycle": "stable",
+    "loom:injected_interfaces": {
+      "net/minecraft/class_4910": ["net/fabricmc/fabric/api/datagen/v1/model/FabricBlockStateModelGenerator"],
+      "net/minecraft/class_4915": ["net/fabricmc/fabric/api/datagen/v1/model/FabricItemModelGenerator"],
+      "net/minecraft/class_4942": ["net/fabricmc/fabric/impl/datagen/FabricModel"]
```

Comment:
Dont use interface injection for impl.

Reply:
Now that I think about it I'm not sure if this should even be an impl interface given its use in the API-provided model builders. Given the intent to also allow `ModelBuilder` to be extended for custom models I may also have to reconsider how to go about this to actually allow for any custom properties that other mods may choose to incorporate.